### PR TITLE
UNO 300 - Upload media manager in passage authoring

### PR DESCRIPTION
**Pull request by habemuscode**
**Description**

Implement possibility to upload media in CKEditor for passages (images, video)

**AC**
- [x] Be able to upload media files in CKEditor
- [x] Resource manager should have only the Assets folder

### CODE REVIEW

File: views/js/qtiCreator/helper/creatorRenderer.js [hunk 1]
```diff
@@ -66,7 +66,8 @@ define([
                         deleteUrl : config.properties.fileDeleteUrl,
                         downloadUrl : config.properties.fileDownloadUrl,
                         fileExistsUrl : config.properties.fileExistsUrl,
-                        mediaSourcesUrl : config.properties.mediaSourcesUrl
+                        path: 'taomedia://mediamanager/',
```
Comment (andrey-ponamarev) [inline views/js/qtiCreator/helper/creatorRenderer.js:69]: How about to move values to the controller properties?

File: views/js/qtiCreator/widgets/static/img/Widget.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        this.registerStates(states);
+
+        Widget.initCreator.call(this);
+
+        inlineHelper.togglePlaceholder(this);
+
+        //check file exists:
+        inlineHelper.checkFileExists(this, 'src', options.baseUrl);
+        $('#item-editor-scope').on(`filedelete.resourcemgr.${this.element.serial}`, (e, src) => {
+            if (self.getAssetManager().resolve(img.attr('src')) === this.getAssetManager().resolve(src)) {
```
Comment (andrey-ponamarev) [inline views/js/qtiCreator/widgets/static/img/Widget.js:44]: Looks like `self` is not declared

File: views/js/qtiCreator/widgets/static/img/Widget.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    ImgWidget.destroy = function destroy(){
+        $('#item-editor-scope').off(`.${this.element.serial}`);
+    };
+
+    ImgWidget.getRequiredOptions = function getRequiredOptions(){
+        return ['baseUrl', 'uri', 'lang', 'mediaManager', 'assetManager'];
+    };
+
+    ImgWidget.buildContainer = function buildContainer(){
+
```
Comment (andrey-ponamarev) [inline views/js/qtiCreator/widgets/static/img/Widget.js:60]: Please, use prettier for code formatting https://github.com/oat-sa/prettier-config

File: views/js/qtiCreator/widgets/static/img/Widget.js (hunk 2 above)
Comment (bziondik) [reply views/js/qtiCreator/widgets/static/img/Widget.js]: fixed

**Pull request merged**